frontend/src/components/ShootHibernation/ManageHibernationSchedule.vue

```diff
@@ -253,6 +260,9 @@ export default {
       this.purpose = purpose
       this.parseSchedules(hibernationSchedule)
       this.setNoHibernationSchedule(noHibernationSchedule)
+      this.$nextTick(() => {
```

Comment:
Why do we need this $nextTick? Add a comment that explains why it is needed.

Reply:
added comment